cert-manager/config/rbac/service_account.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/name: serviceaccount
+    app.kuberentes.io/instance: cert-manager
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: cert-manager
+    app.kubernetes.io/part-of: cert-manager
+    app.kubernetes.io/managed-by: kustomize
+  name: cert-manager
```

Comment:
maybe a different name would be good to work against ambiguities with the official cert-manager